packages/protected-data-delivery-dapp/deployment/abis/DataProtectorSharingABI.json

```diff
@@ -15,6 +15,11 @@
         "internalType": "contract AddOnlyAppWhitelistRegistry",
         "name": "addOnlyAppWhitelistRegistry_",
         "type": "address"
+      },
+      {
+        "internalType": "contract IVoucherHub",
+        "name": "_voucherHub",
+        "type": "address"
```

Comment:
Just to understand: Not related to this PR, right?
This should be in the voucher branch?
Is this generated code?